deform/templates/sequence_item.pt

```diff
@@ -20,11 +20,11 @@
   </div>
   <div class="form-group col-1" style="padding:0">
     <!-- sequence_item -->
-    <span class="deform-order-button close glyphicon glyphicon-resize-vertical"
-          id="${oid}-order"
-          tal:condition="not hidden"
-          title="Reorder (via drag and drop)"
-          i18n:attributes="title"></span>
+    <a class="deform-order-button close"
+       id="${oid}-order"
+       tal:condition="not hidden"
+       title="Reorder (via drag and drop)"
+       i18n:attributes="title">&udarr;</a>
```

Comment:
This is closer to the original.

```suggestion
       i18n:attributes="title">&updownarrow;</a>
```

Also changing from a `span` to `a` might cause some issues—with its display, usability, or accessibility—maybe not. I have not verified.